packages/web-components/src/components/leaving-ibm/leaving-ibm-composite.ts

```diff
@@ -74,6 +77,36 @@ class DDSLeavingIbmComposite extends ModalRenderMixin(LitElement) {
   @property({ reflect: true })
   href = '';
 
+  @HostListener('document:click')
+  protected _handleDocumentClick = (event: PointerEvent): void => {
+    if (!this.open) {
+      const { target } = event;
+      const linkTarget =
+        target instanceof HTMLAnchorElement
+          ? target
+          : (event.composedPath().find(pathTarget => {
+              return pathTarget instanceof HTMLAnchorElement;
+            }) as HTMLAnchorElement | undefined);
+
+      if (linkTarget) {
+        const linkIsExternal = linkTarget.hasAttribute('data-external');
```

Comment:
Does anyone have any opinions on the name of this attribute? `'data-external'` feels a bit too generic. Maybe `'data-leaving-ibm'`?